cells.py

```diff
@@ -173,31 +176,31 @@ def setupBoard(self):
             i.makePrisoners()
 
     def resetGame(self):
-         already_reset = True
-         for cell in self.cells:
-             if len(cell.prisoners) != 2:
-                 already_reset = False
-         if not already_reset:
-             self.move_count += 1
-             for cell in self.cells:
-                 cell.reset()
-       
+        already_reset = True
+        for cell in self.cells:
+            if len(cell.prisoners) != 2:
+                already_reset = False
+        if not already_reset:
+            self.move_count += 1
+            for cell in self.cells:
+                cell.reset()
+
     def gameloop(self):
         self.setupBoard()
         playing = True
         while playing:
             self.clock.tick(self.fps)
 
-            text1 = Text(str(self.move_count), 
-                        size = int(35 * self.scale), 
-                        color = black)
+            text1 = Text(str(self.move_count),
+                         size=int(35 * self.scale),
+                         color=black)
             text1.rect.center = (int(600 * self.scale), int(450 * self.scale))
-            text2 = Text(_("(h)elp"), size = int(50 * self.scale))
+            text2 = Text(_("(h)elp"), size=int(50 * self.scale))
             text2.rect.topleft = (int(10 * self.scale), int(10 * self.scale))
             self.text = Group((text1, text2))
 
             if (len(self.escArea.prisoners.sprites()) == self.cell_count
-                and self.guard.moving == False):
+               and not self.guard.moving):
```

Comment:
The indentation here looks wrong.

Reply:
Fixed, but introduced an flake8 issue, but the line is just 81 characters.